jenkins-master/jobs/mozilla-aurora_functional/config.xml

```diff
@@ -195,7 +174,6 @@ ${FAILED_TESTS}
       <timeoutType>absolute</timeoutType>
       <timeoutMinutesElasticDefault>3</timeoutMinutesElasticDefault>
     </hudson.plugins.build__timeout.BuildTimeoutWrapper>
-    <hudson.plugins.timestamper.TimestamperBuildWrapper plugin="timestamper@1.5.14"/>
```

Comment:
Are we no longer outputting timestamps in the build log because our logs will have them when they are uploaded?